bin/gpu_usage_chart

```diff
@@ -1,11 +1,5 @@
 #!/usr/bin/env python
-
-import sys
-
-import os
-sys.path.append(os.path.abspath('.'))
-
-from util.gpu_usage import GPUUsage
+from deepspeech_training.util.gpu_usage import GPUUsage
```

Comment:
Maybe we can remove those `gpu_usage_chart` / `gpu_usage_plot`, I don't think they are useful anymore?

Reply:
I've never used them. If you're OK with removing I can do that.

Reply:
It's been years since I used it, so just remove :)